packages/@lightningjs/ui-components/src/components/Icon/Icon.js

```diff
@@ -20,87 +20,156 @@ import lng from '@lightningjs/core';
 import Base from '../Base';
 import * as styles from './Icon.styles.js';
 import { context } from '../../globals';
-import { stringifyCompare, getValidColor } from '../../utils';
+import { getValidColor } from '../../utils';
 import CustomImageTexture from '../../textures/CustomImageTexture';
 
+/**
+ * Icon component that displays different types of icons, supporting SVGs and images.
+ * @extends Base
+ */
```

Comment:
is the plan to start including these jsdoc blocks for methods in all classes? i dont have a problem with them, just asking from a code standards/style guide standpoint

Reply:
I would like to, but yes good callout. We probably need to decide as a team what the standards are.

Reply:
yeah - i dont want to hold up this PR so it's fine, but we should have a conversation around how we'd want to do this, and also make sure we aren't conflicting with the TS files because I'm pretty sure VSCode will pick these up

Reply:
The potential TS collision is an extremely good point @anthony9187